library/analytics/src/main/java/org/cru/godtools/analytics/AppSeeAnalyticService.java

```diff
@@ -0,0 +1,111 @@
+package org.cru.godtools.analytics;
+
+import android.app.Activity;
+import android.app.Application;
+import android.content.Context;
+import android.os.Bundle;
+
+import com.appsee.Appsee;
+import com.appsee.AppseeListener;
+import com.appsee.AppseeScreenDetectedInfo;
+import com.appsee.AppseeSessionEndedInfo;
+import com.appsee.AppseeSessionEndingInfo;
+import com.appsee.AppseeSessionStartedInfo;
+import com.appsee.AppseeSessionStartingInfo;
+import com.crashlytics.android.Crashlytics;
+
+import org.cru.godtools.analytics.model.AnalyticsActionEvent;
+import org.cru.godtools.analytics.model.AnalyticsScreenEvent;
+import org.greenrobot.eventbus.EventBus;
+import org.greenrobot.eventbus.Subscribe;
+import org.greenrobot.eventbus.ThreadMode;
+
+import java.util.Map;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.UiThread;
+
+public class AppSeeAnalyticService implements AnalyticsService, Application.ActivityLifecycleCallbacks, AppseeListener {
+
+    @Nullable
+    private static AppSeeAnalyticService sInstance;
+
+    private AppSeeAnalyticService(Context context) {
+        EventBus.getDefault().register(this);
+        ((Application) context.getApplicationContext()).registerActivityLifecycleCallbacks(this);
+        Appsee.addAppseeListener(this);
+    }
+
+    @NonNull
+    public static synchronized AppSeeAnalyticService getInstance(Context context) {
```

Comment:
change this method to `start` and have it take an `Application` instead of `Context`

Reply:
only reason I'm saying `start` vs `getInstance` is because this service is just being started, we never interact directly with our singleton